api/app-config.yaml

```diff
@@ -1,3 +1,6 @@
+auth:
+  oidc_client_id: "{{.OIDC_CLIENT_ID}}"
+  oidc_issuer_url: "{{.OIDC_ISSUER_URL}}"
```

Comment:
this is the notation we can use to drop in environment variables to our configuration files